packages/user-ui/src/pages/main/mod.rs

```diff
@@ -1,5 +1,5 @@
 mod controller;
-mod i18n;
+pub mod i18n;
```

Comment:
I do not recommend expose i18n to higher module.
Why do you need it? instead of it, I recommend i18n is placed in same module of a component.